test/k8sT/PoliciesNightly.go

```diff
@@ -58,15 +66,28 @@ var _ = Describe("NightlyPolicies", func() {
 		// Delete all pods created
 		kubectl.Exec(fmt.Sprintf(
 			"%s delete --all pods,svc,cnp -n %s", helpers.KubectlCmd, helpers.DefaultNamespace))
-
+		_ = kubectl.HeapsterDelete()
```

Comment:
no need for the `_ =`

Reply:
@aanm I want this done in the tests to ensure that we do not accidentally ignore the results of certain commands which we should be checking results of, which has previously been a source of issues in the testing framework. As such, when we do want to ignore errors, I prefer we explicitly specify we are ignoring them, which is the case in `AfterFailed`, and other `After` Ginkgo functions.